### PR TITLE
requestSignatureVerification property added to Application Page

### PR DESCRIPTION
**Pull request by rahul-nagraj**

### CODE REVIEW

File: changelog/Microsoft.DirectoryServices.json [hunk 1]
```diff
@@ -186,6 +186,40 @@
...
       "SubArea": "Identity and sign-in"
     },
+    {
+      "ChangeList": [
+        {
+          "Id": "dbd62dae-abc1-419d-8322-045d266f0b45",
+          "ApiChange": "Enumeration",
+          "ChangedApiName": "weakAlgorithms",
+          "ChangeType": "Addition",
+          "Description": "Added the [weakAlgorithms](https://docs.microsoft.com/en-us/?view=graph-rest-beta) resource",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:196]: Fixing a stub generator issue
```suggestion
          "Description": "Added the [weakAlgorithms](https://docs.microsoft.com/en-us/?view=graph-rest-beta) enumeration type.",
```

File: changelog/Microsoft.DirectoryServices.json [hunk 2]
```diff
@@ -186,6 +186,40 @@
...
+          "Target": "weakAlgorithms"
+        },
+        {
+          "Id": "dbd62dae-abc1-419d-8322-045d266f0b45",
+          "ApiChange": "Resource",
+          "ChangedApiName": "requestSignatureVerification",
+          "ChangeType": "Addition",
+          "Description": "Added the [requestSignatureVerification](https://docs.microsoft.com/en-us/graph/api/resources/requestSignatureVerification?view=graph-rest-beta) resource",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:204]:
```suggestion
          "Description": "Added the [requestSignatureVerification](https://docs.microsoft.com/en-us/graph/api/resources/requestSignatureVerification?view=graph-rest-beta) resource.",
```

File: changelog/Microsoft.DirectoryServices.json [hunk 3]
```diff
@@ -186,6 +186,40 @@
...
+          "Target": "requestSignatureVerification"
+        },
+        {
+          "Id": "dbd62dae-abc1-419d-8322-045d266f0b45",
+          "ApiChange": "Property",
+          "ChangedApiName": "requestSignatureVerification",
+          "ChangeType": "Addition",
+          "Description": "Added the [requestSignatureVerification](https://docs.microsoft.com/en-us/graph/api/resources/application?view=graph-rest-beta) resource",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:212]: Fixing a stub generator issue
```suggestion
          "Description": "Added the **requestSignatureVerification** property to the [application](https://docs.microsoft.com/en-us/graph/api/resources/application?view=graph-rest-beta) resource.",
```

File: changelog/Microsoft.DirectoryServices.json [hunk 4]
```diff
@@ -186,6 +186,40 @@
...
+          "Target": "application"
+        }
+      ],
+      "Id": "dbd62dae-abc1-419d-8322-045d266f0b45",
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2022-07-22T16:03:39.5633059Z",
+      "WorkloadArea": "Identity and access",
+      "SubArea": "Directory management"
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:221]:
```suggestion
      "WorkloadArea": "Applications",
      "SubArea": ""
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @rahul-nagraj - I've left comments and suggestions for minor issues to address.

**Comment by rahul-nagraj** [comment]
> Hi @rahul-nagraj - I've left comments and suggestions for minor issues to address.

Hi @FaithOmbongi, I have addressed all the review comments. Let me know if this looks good now

**Comment by rahul-nagraj** [comment]
#sign-off

**Review by FaithOmbongi — APPROVED** [review]

**Pull request merged**